Pipelines/Templates/set-environment-id.yml

```diff
@@ -3,12 +3,11 @@ parameters:
   type: string
 
 steps:
-- powershell: |  
-    $microsoftXrmDataPowerShellModule = '$(CoETools_Microsoft_Xrm_Data_PowerShell)'
-    Import-Module $microsoftXrmDataPowerShellModule
-    $conn = Get-CrmConnection -ConnectionString "$(CdsBaseConnectionString)${{parameters.serviceConnection}}"
-    # Get the Environment name (which is a GUID)
-    $environmentId = $conn.EnvironmentId 
-    # Set the EnvironmentId as a global variable for use in other templates
-    echo "##vso[task.setvariable variable=EnvironmentId]$environmentId"
-  displayName: 'Set EnvironmentId'
+- task: microsoft-IsvExpTools.PowerPlatform-BuildTools.tool-installer.PowerPlatformToolInstaller@2
+  displayName: 'Install Power Platform Build Tools'

```

Comment:
Is this necessary? We install build tools as one of the first tasks of every pipeline template.

Reply:
@jenschristianschroder Removed the "PowerPlatformToolInstaller" task.